metals/src/main/scala/scala/meta/internal/metals/JsonParser.scala

```diff
@@ -0,0 +1,35 @@
+package scala.meta.internal.metals
+import com.google.gson.{Gson, JsonElement, JsonObject}
```

Comment:
Let's keep separate line imports, otherwise it's a bit of a mess in the project. Let's keep it consistent